.github/workflows/ansible-test.yml

```diff
@@ -153,37 +113,18 @@ jobs:
             python: 3.9
 
     steps:
-      - name: Check out code
-        uses: actions/checkout@v2
-        with:
-          path: ansible_collections/${{env.NAMESPACE}}/${{env.COLLECTION_NAME}}
-
-      - name: Set up Python
-        uses: actions/setup-python@v2
-        with:
-          # it is just required to run that once as "ansible-test integration" in the docker image
-          # will run on all python versions it supports.
-          python-version: 3.8
-
-      - name: Install ansible-base (${{ matrix.ansible }})
-        run: pip install https://github.com/ansible/ansible/archive/${{ matrix.ansible }}.tar.gz --disable-pip-version-check
-
-      # OPTIONAL If your integration test requires Python libraries or modules from other collections
-      # Install them like this
-      - name: Install collection dependencies
-        run: ansible-galaxy collection install ansible.netcommon -p .
-
-      # Run the integration tests
-      - name: Run integration test
-        run: ansible-test integration -v --color --retry-on-error --continue-on-error --diff --python ${{ matrix.python }} --docker --coverage
-        working-directory: ./ansible_collections/${{env.NAMESPACE}}/${{env.COLLECTION_NAME}}
-
-        # ansible-test support producing code coverage date
-      - name: Generate coverage report
-        run: ansible-test coverage xml -v --requirements --group-by command --group-by version
-        working-directory: ./ansible_collections/${{env.NAMESPACE}}/${{env.COLLECTION_NAME}}
-
-      # See the reports at https://codecov.io/gh/GITHUBORG/REPONAME
-      - uses: codecov/codecov-action@v1
+      - name: >-
+          Perform integration testing against
+          Ansible version ${{ matrix.ansible }}
+          under Python ${{ matrix.python }}
+        uses: ansible-community/ansible-test-gh-action@release/v1
         with:
-          fail_ci_if_error: false
+          ansible-core-version: ${{ matrix.ansible }}
+          # OPTIONAL command to run before invoking `ansible-test integration`
+          # pre-test-cmd:
+          python-version: 3.9
+          target-python-version: ${{ matrix.python }}
+          testing-type: integration
+          # OPTIONAL If your integration tests require Python libraries
+          # or modules from other collections, install them like this
+          test-deps: ansible.netcommon
```

Comment:
How does this allow to install Python libraries? The examples in this workflow only show how to install collection dependencies. (Also I'm not sure whether this makes sense at all, since libraries don't help at all, since the tests run inside a Docker container?)

Reply:
There's no way to specify Python libraries at the moment. And I also don't know if it's necessary. I guess, we could think about it when designing the v2 API that would support newly added `ansible-test` args. I wonder if there's space for supporting this via execution environments that already support pip/OS/collection packages via a standardized config. Or at least use the same config across all tools (we already want this to be the case for navigator, vscode extension and ALS anyway).
But it's a question for the future because we still haven't gotten ownership of `ansible-test`.

Reply:
I don't think it's necessary, since ansible-test already allows you to install Python libraries directly in the containers (with the `tests/unit/requirements.txt` / `tests/integration/requirements.txt` files). I was mainly asking because the comment explicitly mentions Python libraries. I would rather remove that comment than add support for this.